tests/pyfunc/test_responses_agent.py

```diff
@@ -1192,3 +1197,266 @@ def test_create_function_call_output_item():
 def test_prep_msgs_for_cc_llm(responses_input, cc_msgs):
     result = ResponsesAgent.prep_msgs_for_cc_llm(responses_input)
     assert result == cc_msgs
+
+
+@pytest.mark.skipif(not _HAS_LANGCHAIN_BASE_MESSAGE, reason="LangChain is not installed")
```

Comment:
Can we always make sure this is tested? If it's depend on certain langchain version then let's check langchain version instead